hw4/Task_3.ts

```diff
@@ -0,0 +1,15 @@
+//Task 3
+// Дана произвольная строка. Реализуйте метод, который принимает позицию символа в строке.
+// Если переданное число больше длины строки, то должно отобразиться сообщение:
+// `Вы вышли за границу строки`
+
+function myFunction3(cat: string): string {
```

Comment:
Please pay attention to naming of the functions and variables. They have to be more explicit and sensible.
For instance "myFunction3" -> "checkStingLength", "cat" -> "sentence".

Reply:
Ok, will take in mind for the future.
Fixed.